AnKi/Core/ConfigVars.defs.h

```diff
@@ -16,6 +16,7 @@ ANKI_CONFIG_VAR_BOOL(CoreMaliHwCounters, false, "Enable Mali counters")
 ANKI_CONFIG_VAR_U32(Width, 1920, 16, 16 * 1024, "Width")
 ANKI_CONFIG_VAR_U32(Height, 1080, 16, 16 * 1024, "Height")
 ANKI_CONFIG_VAR_BOOL(WindowFullscreen, false, "Start at fullscreen")
+ANKI_CONFIG_VAR_BOOL(FullscreenExclusive, false, "Use exclusive mode, while at fullscreen")
```

Comment:
I'm thinking that maybe we can have one variable for fullscreen. It can take values 0, 1 and 2. The 1 is borderless and 2 is exclusive.

Reply:
I was feeling same at first, yet decide to go for a split variables due to lack of enum's in config files.
Having syntax like `Sponza.exe WindowFullscreen 2` may not be what we want.

Reply:
Changed to `ANKI_CONFIG_VAR_U32(WindowFullscreen, 1, 0, 2, "Start at fullscreen")`